app/controllers/collection_drafts_controller.rb

```diff
@@ -74,9 +74,7 @@ def update
 
       if gkr_enabled? && params[:recommended_keywords_viewed] == 'true'
         # create keyword_recommendation
-        get_resource.record_recommendation_provided
-
-        # TODO: if/else and logging
+        get_resource.record_recommendation_provided(params[:gkr_request_id], params[:recommended_keyword_list])
```

Comment:
Do we want to record the recommendation when we error?

Reply:
This is code from an early commit.

Reply:
No, it shouldn't be recorded.  I added a guard in set_keyword_recommendations to ensure that the hidden params would not be generated in the descriptive_keywords form.